server/claim/types/sentence/sentence.service.ts

```diff
@@ -123,7 +111,45 @@ export class SentenceService {
                     },
                 },
             },
-        ]);
+        ];
+
+        if (searchText) {
+            pipeline.splice(0, 0, {
+                $search: {
+                    index: "sentence_fields",
+                    text: {
+                        query: searchText,
+                        path: "content",
+                        fuzzy: {
+                            maxEdits: 2,
+                        },
+                    },
+                },
+            });
+        } else {
+            pipeline.splice(0, 0, {
+                $search: {
+                    index: "sentence_fields",
+                    text: {
+                        query: filter,
+                        path: "topics",
+                        fuzzy: {
+                            maxEdits: 2,
+                        },
+                    },
+                },
+            });
+        }
+
+        if (filter) {
```

Comment:
question: what happens if searchText is provided without filter? Also, does the order matter?

Reply:
Is gonna bring a common search with all results based on the searchText. And about the order of what specific, the splice on the pipe line or about the searchText and filter ?